### PR TITLE
[v2.9] Backport PR 271: Index more sqlite cache fields.

### PR DESCRIPTION
**Pull request by ericpromislow**

### CODE REVIEW

**Review by moio — APPROVED** [review]
Patches are semantically identical to the originals apart from dependency versions, which is expected.

#271 :
```
git range-diff 63683abbf478aa2dd6f5cb492f6c75130d6e05e7~..63683abbf478aa2dd6f5cb492f6c75130d6e05e7  06c2eb50d11d4f07f7a690e4bc0f71a805a114ee~..06c2eb50d11d4f07f7a690e4bc0f71a805a114ee
```

#291 :
```
git range-diff 6a11ffb66087599f9ec301eecb44841b0c276b10~..6a11ffb66087599f9ec301eecb44841b0c276b10 7dd4cbefea2f7e35346038e190dd3de12bcd7c8f~..7dd4cbefea2f7e35346038e190dd3de12bcd7c8f
```


LGTM

File: go.mod [hunk 1]
```diff
@@ -24,15 +23,15 @@ require (
 	github.com/rancher/dynamiclistener v0.6.0-rc2
 	github.com/rancher/kubernetes-provider-detector v0.1.5
 	github.com/rancher/lasso v0.0.0-20240923125127-ae858d002589
-	github.com/rancher/norman v0.0.0-20240708202514-a0127673d1b9
+	github.com/rancher/norman v0.0.0-20240822182819-60ccfabc4ac5
```
Comment (tomleb) [inline go.mod:26]: This points to a commit of norman on the main branch, not the `release/v2.9` branch. Any reason why this is needed?

**Review by tomleb — CHANGES_REQUESTED** [review]
I would suggest just backporting the PR from 271 and not the uber bump as well. The uber change could cause problem or major changes when bumping steve in rancher/rancher. eg: Quite a few changes (although trivial): https://github.com/rancher/rancher/pull/47128.

We're gonna be in code freeze by EOD so let's minimize the impact and just go with the 271 backport (at least for this PR)

**Comment by moio** [comment]
https://github.com/rancher/steve/pull/291 removed

**Review by moio — APPROVED** [review]

**Comment by ericpromislow** [comment]
Validation failed for me without #291, but I see it's passing CI. OK...

**Review by tomleb — APPROVED** [review]
Seems like a clean backport of https://github.com/rancher/steve/pull/271

> Validation failed for me without https://github.com/rancher/steve/pull/291, but I see it's passing CI. OK...

that's most likely due to wrangler moving to the uber dependency. Since we aren't updating wrangler here, no issue.

**Pull request merged**